src/snowflake/snowpark/dataframe.py

```diff
@@ -4304,6 +4306,23 @@ def dtypes(self) -> List[Tuple[str, str]]:
         ]
         return dtypes
 
+    def toJSON(self) -> "DataFrame":
+        """
+        Converts each row of the DataFrame into a JSON string.
```

Comment:
don't we also have to handle `_emit_ast` here?

Reply:
As far as I can tell none of the other dataframe functions handle _emit_ast yet so I assume this does not need to either.